fe/fe-core/src/main/java/com/starrocks/statistic/HyperStatisticsCollectJob.java

```diff
@@ -190,4 +190,9 @@ public String toString() {
                 ", properties=" + properties +
                 '}';
     }
+
+    @Override
+    public String getName() {
```

Comment:
you could distinguish between full and sample according to analyze type in hyper collect job

Reply:
fixed